tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1013,14 +1013,14 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'FP32 model weights and biases are compressed to FP16. '
                                    'All intermediate data is kept in original precision.',
                               choices=["FP16", "FP32", "half", "float"],
-                              default='float',
+                              default='FP16',
                               action=DeprecatedOptionCommon)
     common_group.add_argument('--compress_to_fp16',
                               help=mo_convert_params_common['compress_to_fp16'].description,
                               type=check_bool,
                               nargs="?",
                               const=True,
-                              default=False)
+                              default=None)
```

Comment:
**comment for reviewers:** If we set here to `True`, we will loose information whether we got `True` from the default or user specified it explicitly, and without this information we cannot decide should we print him this warning or no. Therefore set here to `None`, and rewrite to default subsequently: 
https://github.com/pavel-esir/openvino/blob/ae1f9d1f9cb11e5ceb433c6fae9e2c60743b4d35/tools/mo/openvino/tools/mo/convert_impl.py#L219-L224

Reply:
Let us have `True` value here anyway. 
If he/she even specifies `--compress_to_fp16=True`, we should anyway print this message about possible accuracy drop. This is not bad approach at all.